client/components/tile-grid/style.scss

```diff
@@ -90,6 +94,7 @@
 			border-top: 2px solid lighten( $gray, 20% );
 			border-right: 2px solid lighten( $gray, 20% );
 			transform: rotate(45deg);
+			color: #fff;
```

Comment:
Can we make this `$white`

Reply:
🤦‍♂️

Reply:
Fixed in 767901dbaa7d15a800ddb58384cd9d08744af165.